internal/services/r2_bucket/testdata/r2bucketbasic.tf

```diff
@@ -1,6 +1,7 @@
 
   resource "cloudflare_r2_bucket" "%[1]s" {
-    account_id = "%[2]s"
-    name       = "%[1]s"
-    location   = "ENAM"
+    account_id    = "%[2]s"
+    name          = "%[1]s"
+    location      = "ENAM"
+    storage_class = "Standard"
```

Comment:
given the storage class is computed and optional, we should have a test case where it isn't supplied to cover that scenario (like `basic.tf` did)

Reply:
`r2bucketminimum.tf` should cover this, no?

Reply:
looks like we had duplicates in `r2bucketminimum.tf` and `minimum.tf`.